Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -237,15 +237,16 @@ function vtkRenderWindowInteractor(publicAPI, model) {
     });
     container.addEventListener('pointerup', publicAPI.handlePointerUp);
     container.addEventListener('pointercancel', publicAPI.handlePointerCancel);
-    document.addEventListener('keypress', publicAPI.handleKeyPress);
-    document.addEventListener('keydown', publicAPI.handleKeyDown);
-    document.addEventListener('keyup', publicAPI.handleKeyUp);
+    container.addEventListener('keypress', publicAPI.handleKeyPress);
+    container.addEventListener('keydown', publicAPI.handleKeyDown);
+    container.addEventListener('keyup', publicAPI.handleKeyUp);
```

Comment:
It might be better to listen for keyup on the document in the case that the focus leaves the container while a key is still down.

Reply:
I understand your concern, I find your suggestion a bit "fragile" and prone to further errors. 
Can we wait to have a concrete example when there such case happen to see how to best fix it ?

Reply:
I'm not sure what issues would arise, unless there is a "keyup" operation that isn't cleaning up from a "keydown" event, or the "keyup" listener is not a no-op for viewports that are not focused. I just see the one case where someone expects start/end semantics with keydown/keyup, as there is no concept of "keyleave".

Realistically, I don't think it affects our codebases at the moment. I've always preferred handling keypresses at the application level rather than at the vtk.js level. I'm fine keeping it as-is if there are tangible issues with listening for keyup on the document.